pypesto/objective/amici/amici.py

```diff
@@ -615,6 +615,8 @@ def check_gradients_match_finite_differences(
         if x is None and 'petab_problem' in dir(self.amici_object_builder):
             x = self.amici_object_builder.petab_problem.x_nominal_scaled
             x_free = self.amici_object_builder.petab_problem.x_free_indices
-        return super().check_gradients_match_finite_differences(
-            x=x, x_free=x_free, *args, **kwargs
+        return (
```

Comment:
where is  this `_ObjectiveBase` here coming from?

Reply:
[Name mangling (`__name` attributes)](https://docs.python.org/3/tutorial/classes.html#private-variables)